java/code/src/com/redhat/rhn/domain/server/MinionServer.java

```diff
@@ -166,9 +166,12 @@ public boolean doesOsSupportsTransactionalUpdate() {
 
     @Override
     public boolean doesOsSupportsMonitoring() {
-        return isSLES12() || isSLES15() || isLeap15() || isUbuntu1804() || isUbuntu2004() || isUbuntu2204() ||
-                isRedHat6() || isRedHat7() || isRedHat8() || isRedHat9() || isAlibaba2() || isAmazon2() ||
-                isAmazon2023() || isRocky8() || isRocky9() || isDebian12() || isDebian11() || isDebian10();
+        return isSLES12() || isSLES15() || isLeap15() || isLeapMicro() ||
```

Comment:
Here I see we also included Leap Micro.
While in the rest of the code, the changes seems to be specific for SLE Micro, that's ok?

Reply:
yes, this is correct.